functorch/examples/ensembling/parallel_train.py

```diff
@@ -68,27 +68,22 @@ def forward(self, x):
 
 
 loss_fn = nn.NLLLoss()
-
-# Step 3: Make the model functional(!!) and define a training function.
-# NB: this mechanism doesn't exist in PyTorch today, but we want it to:
-# https://github.com/pytorch/pytorch/issues/49171
-func_model, weights = make_functional(MLPClassifier().to(DEVICE))
-
+model = MLPClassifier().to(DEVICE)
 
 def train_step_fn(weights, batch, targets, lr=0.2):
     def compute_loss(weights, batch, targets):
-        output = func_model(weights, batch)
+        output = torch.nn.utils.stateless.functional_call(model, weights, batch)
```

Comment:
Why isn't this `torch.nn.utils.stateless.functional_call(model, weights, (batch,))` ?

Reply:
Great question! In looking into it, it looks like the functional_call docs are different than the behavior, which assumes that `args` is a single argument if it's not a tuple. I'll make another commit updating those docs since it clearly confused me and updating the subsequent commits here to not wrap into a tuple